lib/livebook_web/live/explore_live.ex

```diff
@@ -101,15 +138,15 @@ defmodule LivebookWeb.ExploreLive do
       <div class="mt-4">
         <ul>
           <%= for {notebook_info, number} <- Enum.with_index(@group_info.notebook_infos, 1) do %>
-            <li class="py-4 flex items-center space-x-5 border-b border-gray-200 last:border-b-0">
+            <li class="py-4 flex sm:flex-row flex-col sm:items-center items-start sm:space-x-5 border-b border-gray-200 last:border-b-0">
```

Comment:
```suggestion
            <li class="py-4 flex flex-col sm:flex-row items-start sm:items-center sm:space-x-5 border-b border-gray-200 last:border-b-0">
```

It's a small detail, but it's easier to follow the classes when they are in size-increasing order, like `something sm:something-else` :)